src/ert/config/queue_config.py

```diff
@@ -46,13 +46,32 @@ class QueueOptions(
 ):
     name: QueueSystem
     max_running: pydantic.NonNegativeInt = 0
-    submit_sleep: pydantic.NonNegativeFloat = 0.0
     num_cpu: pydantic.NonNegativeInt = 1
     realization_memory: pydantic.NonNegativeInt = 0
     job_script: str = shutil.which("fm_dispatch.py") or "fm_dispatch.py"
+    submit_sleep: float | None = None
     project_code: str | None = None
     activate_script: str | None = Field(default=None, validate_default=True)
 
+    @field_validator("submit_sleep", mode="before")
+    @classmethod
+    def validate_submit_sleep(
+        cls, v: float | None, info: ValidationInfo
+    ) -> float | None:
+        if v:
+            ConfigWarning.deprecation_warn(
+                "The SUBMIT_SLEEP keyword in QUEUE OPTIONS is deprecated, "
+                "use the global SUBMIT_SLEEP keyword instead",
+            )
+            logger.info("Using deprecated SUBMIT_SLEEP")
+        if v and float(v) < 0.0:
```

Comment:
Do we need this test here also? Since the same validation also happen when QueueConfig is initialized, that might suffice?

Reply:
This is needed for validation when the submit sleep is part of a queue option ie : QUEUE_OPTION LSF SUBMIT_SLEEP -5.5